networkx/algorithms/connectivity/tests/test_cuts.py

```diff
@@ -34,7 +34,7 @@ def _generate_no_biconnected(max_attempts=50):
 def test_articulation_points():
     Ggen = _generate_no_biconnected()
     for flow_func in flow_funcs:
-        for i in range(3):
+        for i in range(1):
```

Comment:
Thanks for the review!!
I did intend to leave the loop there so it would be easy for someone to change back to multiple random graphs. But perhaps that is misguided on my part.

What do you think?  I could put a comment that the 1 could be increased for testing multiple realizations.